internal/auth/ldap/auth_method.go

```diff
@@ -175,7 +183,7 @@ func (am *AuthMethod) convertUrls(ctx context.Context) ([]any, error) {
 		if err != nil {
 			return nil, errors.Wrap(ctx, err, op)
 		}
-		obj, err := NewUrl(ctx, am.PublicId, priority, parsed)
+		obj, err := NewUrl(ctx, am.PublicId, priority+1, parsed)
```

Comment:
Can you document that priority is incremented by 1 in this function/ why?

Reply:
The `NewUrl(...)` documentation states that `connectionPriority cannot be less than one`.  That isn't sufficient, given that the range index will start at zero?

Reply:
That's fair! I was just thinking it'd be a little less digging for someone in the future